metadata.yaml

```diff
@@ -26,6 +26,10 @@ storage:
   vault:
     type: filesystem
     location: /var/snap/vault/common
+  certs:
+    type: filesystem
+    minimum-size: 5M
```

Comment:
I copied `minimum-size` from the `vault-k8s-operator`, but admittedly I'm not entirely sure what the purpose of it is.

https://github.com/canonical/vault-k8s-operator/blob/e4b36ed49378d3f7632f6711e77a891c7dfd96f6/metadata.yaml#L50